### ISSUE TITLE
"Driver is not defined" when trying to run current master branch.

### ISSUE THREAD

**Issue by RoboPhred** [zwave-js/zwave-js#749]
Due to the mixing of a typed import with typescript's `emitDecoratorMetadata` , the current master branch cannot run.

The problem is probably found in other Command Classes, but the one I am running into is in `MultiChannelCC.ts`.
The use of a typed import
`import type { Driver } from "../driver/Driver";`
does not import Driver class into this file.  However, the MultiChannelCC constructor is trying to use it:
```js
public constructor(driver: Driver, options: CommandClassOptions) 
```

Because it is a type passed to the constructor, `emitDecoratorMetadata` tries to store that type in metadata.  The compiled output is:
```js
MultiChannelCC = __decorate([
    CommandClass_1.commandClass(CommandClasses_1.CommandClasses["Multi Channel"]),
    CommandClass_1.implementedVersion(4),
    __metadata("design:paramtypes", [Driver, Object])
], MultiChannelCC);
```

Driver is attempted to be used here, but the import is a type import, so there is no Driver defined.  This causes a crash when loading the library.

The solution would be to import all constructor arguments as standard imports.

This might be an issue we should bring up with the Typescript devs, as it is a bit silly that it allows you to do this.

Edit: Whatever is running on the typescript sandbox seems to guard against this happening:
[sandbox example](https://www.typescriptlang.org/play/?experimentalDecorators=true&emitDecoratorMetadata=true&target=2&module=1#code/JYWwDg9gTgLgBDAnmApnA3nAYhCAhAQyjgF84AzKCEOAIhQA8DwAbFAWhAgBMBXN2gG4AUMPK8AdgGMYwCBLghEAERRToBGNAAUASgzC4RuFBQxeUBdqSoAXHAITE+gLwA+DKWElRAASWq6lCaOrrCjJCwcFIsBADOcXAAsoh4VADWKBIAwrEJBsbR8nEwULwyOuS49jj4RPqYPj5AA)
Note the typeof check around FooBar.

**Comment by RoboPhred** [comment]
I was able to resolve this locally by shutting off `emitDecoratorMetadata`.  As far as I can tell, zwave-js does not use this design time metadata for anything, so shutting it off should be safe.

**Comment by AlCalzone** [comment]
Ooooohhh, you've found the root case for another issue. `ts-node` only runs in `transpileOnly` mode since TS 3.8.

**Comment by AlCalzone** [comment]
I think its safe to remove that option. I had it on from the start when I was planning to do more decorator magic. 

Raised https://github.com/microsoft/TypeScript/issues/37672 to make the TypeScript team aware of it.

**Comment by AlCalzone** [comment]
fixed by #751
